### PR TITLE
PLAT-959 | Recent terms on content classifier

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+			}"
+		>
+			<span class="icon is-small">
+				<i class="fa fa-tag"></i>
+			</span>
+			Dodaj ostatnio użyte pojęcie: <strong>{{lastUsedTerm.tag.name}}</strong>
+		</div>
+		<div
+			v-if="lastUsedTermsSet"
+			@click="canAddLastUsedTermsSet && lastUsedTermsSet.map(term => $emit('attachTaxonomyTerm', term))"
```
Comment (kvas-damian) [inline resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue]: Currently, we call as many requests as we have terms to assign? Do we want to try this approach or create a separate endpoint for attaching multiple terms to multiple items?
On the other hand, we have a batch endpoint to create implemented. Maybe we could reuse it here?

File: resources/assets/js/components/global/contentClassifier/ContentClassifierEditor.vue [hunk 2]
```diff
@@ -232,6 +245,8 @@ export default {
 					ancestors: term.ancestors || this.getAncestorsById(term.id),
 				};
 				this.$emit('taxonomyTermAttached', termToAdd);
+				contentClassifierStore.set(CONTENT_CLASSIFIER_STORE_KEYS.LAST_TERM, termToAdd);
+				contentClassifierStore.set(CONTENT_CLASSIFIER_STORE_KEYS.ALL_TERMS, this.allTaxonomyTerms);
```
Comment (bognix) [inline resources/assets/js/components/global/contentClassifier/ContentClassifierEditor.vue]: correct me if I'm wrong. This is the list of all taxonomy terms used in the editor?

File: resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+			type: Array,
+			required: true,
+		},
+	},
+	computed: {
+		canAddLastUsedTerm() {
+			return this.lastUsedTerm && !this.items.every(item => item.taxonomyTerms.find(term => term.id === this.lastUsedTerm.id));
+		},
+		canAddLastUsedTermsSet() {
+			return this.lastUsedTermsSet && !this.lastUsedTermsSet.every(lastUsedTerm => this.items.every(item => item.taxonomyTerms.find(term => term.id === lastUsedTerm.id)));
```
Comment (bognix) [inline resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue]: I would abstract this part `this.items.every(item => item.taxonomyTerms.find(term => term.id === lastUsedTerm.id))` to a function. The `lastUserTerm` can be passed as an argument.

File: resources/assets/js/components/global/contentClassifier/ContentClassifierEditor.vue (hunk 2 above)
Comment (kvas-damian) [reply resources/assets/js/components/global/contentClassifier/ContentClassifierEditor.vue]: yes, we use all. We didn't agree on anything so I choose a random option. If you feel there is a better option to choose which terms should be presented, let me know.

Comment (bognix) [reply resources/assets/js/components/global/contentClassifier/ContentClassifierEditor.vue]: the option is fine, the variable name wasn't descriptive enough for me, hence I asked :)

File: resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue (hunk 1 above)
Comment (gitkuba) [reply resources/assets/js/components/global/contentClassifier/ContentClassifierEditorRecentTerms.vue]: Both options soud good to me.

**Pull request merged**